src/main/java/fi/otavanopisto/kuntaapi/server/integrations/management/ManagementTileProvider.java

```diff
@@ -165,31 +165,31 @@ public AttachmentData getTileImageData(OrganizationId organizationId, TileId til
     return null;
   }
 
-  private List<Tile> translateTiles(List<fi.otavanopisto.mwp.client.model.Tile> mwpTiles) {
+  private List<Tile> translateTiles(List<fi.otavanopisto.mwp.client.model.Tile> managementTiles) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "translateTiles" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)